### PR TITLE
Added the link to docs

### PR DESCRIPTION
**Pull request by patilswapnilv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ The command executes with one of the following output messages, indicating the s
 - Optimized Resource Grouping
 - Multi-cluster Application Management
 - Support for Workload Hibernation
+- Open Source
```
Comment (Abhinav-26) [inline README.md]: not required to mention open source

File: README.md [hunk 2]
```diff
@@ -86,6 +87,7 @@ The Helm Dashboard is being used and trusted by enterprises and communities all
 ## 👥 Community & Support
  
 Get updates and chat with project maintainers, contributors, and community members
+- [Documentation](https://docs.devtron.ai/getting-started/install/install-devtron)
```
Comment (Abhinav-26) [inline README.md]: add a line space before pointers and change the hyperlink URL to `https://www.docs.devtron.ai/`. Also, make a full sentence. like -

- Feel free to check out the [Documentation](https://www.docs.devtron.ai/)

File: README.md (hunk 1 above)
Comment (patilswapnilv) [reply README.md]: Removed `- Open Source` as requested.

File: README.md (hunk 2 above)
Comment (patilswapnilv) [reply README.md]: I updated the hyperlink as suggested.

**Review by verma-kunal — APPROVED** [review]
Lgtm 🚀

**Pull request merged**